releasenotes/notes/add-logging-component-508126fd23d77e4d.yaml

```
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Change the logging format to include the name of the agent logging
```

```suggestion
    Change the logging format to include the name of the logging agent.
```
Wasn't this included previously? We had `[TRACE]`, `[CORE]`, etc. What's the actual change? Or were some missing. Sorry if I missed something here.

---

This was present in the agent container yes due to the s6 filtering, I should probably rephrase 🤔